docs/language/src/builtins/elements.md

```diff
@@ -143,6 +143,8 @@ or it will be mapped to a private unicode character. The mapping of these non-pr
 
 -   **`has-focus`** (_out_ _bool_): Is `true` when the element has keyboard
     focus.
+-   **`enabled`** (_in_ _bool_): When true, the `FocusScope` will make itself the focused element when clicked. Set this to false if you don't want the click-to-focus
+    behavior. A parent `FocusScope` will still receive key events from child `FocusScope`s that were rejected, even if `enabled` is set to false. (default value: true)
```

Comment:
This will also affect the tab to focus iirc.

> A parent `FocusScope` will still receive key events from child `FocusScope`s that were rejected, even if `enabled` is set to false. (default value: true)

Is that really a behavior we want?

Reply:
> This will also affect the tab to focus iirc.

Ah yes, that's true. A disabled `FocusScope` doesn't accept the tab focus. I'll add that to the docs.

> Is that really a behavior we want?

I think Jocelyn's use-case is valid, and the current code supports that use-case (this PR merely documents). It could be implemented differently, if we deprecated `enabled` in favor of a different property that made the more fine-grained behavior explicit - like a policy.

Do you have a suggestion for an alternative API that could be used? Are you in favor of changing the current behavior?

Reply:
I'm not sure what exactly is the use case.
A focus policy would make sense.
But to me, enabled is a all or nothing kind of flag,  and the logic would be that enabled even propagates to all child, so that if one element is disabled, all children are disabled recursively. (this is not yet implemented but that's how i imagine enabled property should work)

Reply:
Ok, how about I remove the documentation from this PR again - so that we have a little more freedom to make changes to `enabled`, but keep the test and (importantly) the change to `input_event`?